### PR TITLE
build: consolidate production build dependencies

### PR DESCRIPTION
**Pull request by 0-vortex**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [x] 🔖 Release
- [ ] 🚩 Other

## Description

This PR enforces the production lock file for node applications.

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by 0-vortex** [comment]
@bdougie looks clear!

File: astro.config.mjs [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default {
     devOptions: {
         hostname: 'localhost',
         port: 3000,
-        tailwindConfig: './tailwind.config.csj',
+        tailwindConfig: './tailwind.config.cjs',
```
Comment (bdougie) [inline astro.config.mjs:13]: good catch. Was this causing failures anywehre?

**Review by bdougie — APPROVED** [review]
LGTM

File: astro.config.mjs (hunk 1 above)
Comment (0-vortex) [reply astro.config.mjs:13]: no, it wasn't reading the config and defaulting everything AFAIK

**Pull request merged**